docs/tutorials/10_checkpointer_policysaver_tutorial.ipynb

```diff
@@ -373,8 +373,7 @@
         "def train_one_iteration():\n",
         "\n",
         "  # Collect a few steps using collect_policy and save to the replay buffer.\n",
-        "  for _ in range(collect_steps_per_iteration):\n",
```

Comment:
Thank you for the PR and finding this issue! I think instead of removing the for loop we should change the range to be over `num_iterations`, i.e. we still want to run `num_iterations` iterations each collecting `collect_steps_per_iteration` steps. Does that make sense?

Reply:
I have a question. Do we want to run `num_iterations` iterations each collecting `collect_steps_per_iteration` steps within **one training iteration** ? [ These codes are within the function `def train_one_iteration()` ].
Now, if we do, then 'yes' it makes sense. Otherwise, no.

Reply:
I'm very sorry about missing your reply earlier! You are right, since this is running in one training iteration (num_iterations) iterates over train steps, this should be just `collect_driver.run()` like you have.

Reply:
Thank you @egonina  and @tfboyd.